ocaml-lsp-server/src/ocaml_lsp_server.ml

```diff
@@ -35,7 +35,10 @@ let initializeInfo : InitializeResult.t =
       ~referencesProvider:(`Bool true) ~documentHighlightProvider:(`Bool true)
       ~documentFormattingProvider:(`Bool true)
       ~selectionRangeProvider:(`Bool true) ~documentSymbolProvider:(`Bool true)
-      ~renameProvider:(`Bool true) ()
+      ~renameProvider:
+        (`RenameOptions
+          { prepareProvider = Some true; workDoneProgress = None })
```

Comment:
Please use the constructor for RenameOptions. This way, our code will not be broken once new fields are added

Reply:
For unknown reasons `module RenameOptions` is not included in the `Import` module and it is failing to compile:
![Screenshot from 2020-03-18 17-35-28](https://user-images.githubusercontent.com/13620400/76941178-e8a15f00-693e-11ea-8027-dae71e06d8c2.png)

Reply:
It's not just in the `ocaml-lsp-server/src/import.ml`. It is inconvenient for me to maintain it by hand btw.